releasenotes/notes/fix_aqgd_max_grouped_evals-fbe108c005a9b7ac.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    It has been fixed that the AQGD optimizer groups objective function calls by default.
+    The number of grouped evaluations can be controlled via the max_grouped_evals parameter.
```

Comment:
Maybe we can elaborate on this so that someone reading the release note, not knowing the issue, things are clear. Something like this perhaps - feel free to re-word etc.
```
It has been fixed that the AQGD optimizer groups objective function calls by default so that now
it passed a single point to the objective function. For algorithms that can handle a more than
one point in their objective function, such as a VQE in the algorithms here, the number of grouped
evaluations can be controlled via the max_grouped_evals parameter. Grouping evals allows a list
of points to be passed so that potentially they may be more efficiently evaluated in a single job.
```

Reply:
Makes sense. I will do so.